tutor4points/core/templates/requests.html

```diff
@@ -14,14 +14,26 @@
 
 {% block content %}
 
+<div class="show-accepted">
+    <form method="post">
+    {% csrf_token %}
+        <div class="form-check">
+            <input class="form-check-input" type="checkbox" value="" id="show-accepted" name="show-accepted"
+                onChange="this.form.submit()" {{accept_filter}}>
+            <label class="form-check-label" for="show-accepted">
+                Show Accepted
```

Comment:
I think it'll be helpful to have 2 checkboxes as discussed in our last team meeting.  One for the "inbox" and one for the "sent".